src/components/diagrams/singleLineDiagram/single-line-diagram-content.tsx

```diff
@@ -316,6 +365,7 @@ function SingleLineDiagramContent(props) {
     const displayBranchMenu = () => {
         return (
             equipmentMenu.display &&
+            equipmentMenu.equipmentType &&
```

Comment:
the the ligne below we check if equipmentMenu.equipmentType value is contained in the array.
and because the array does not contain 'null' or undefined' the includes function will return false anyway.
the main reason i changend this code is because typescript compiler is complaining about it